src/main.c

```diff
@@ -1,13 +1,133 @@
 #include "../include/permutation.h"
+#include "../include/strconv.h"
+#include "../include/errors.h"
+
+#include <stdio.h>
+#include <stdbool.h>
+#include <getopt.h>
+#include <string.h>
+
+#define HELP                0
```

Comment:
astea poate le-am putea muta in errors.h

Reply:
@deian-balutoiu asta voiam sa fac, vad ca am uitat. O sa le mut azi.